boilerplates/react-union-boilerplate-liferay-basic/liferay/content-tag-portlet/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue May 15 10:31:15 CEST 2018
```

Comment:
Placing comment here, because cannot comment binary, but boilerplates/react-union-boilerplate-liferay-basic/liferay/content-tag-portlet/gradle/wrapper/gradle-wrapper.jar should not be in repo, is there any other way how to do it.?

Reply:
Build doesn't work after gradle-wrapper.jar was deleted. Probably it has to be commited.